Sources/SwiftDocC/Semantics/Metadata/CallToAction.swift

```diff
@@ -15,19 +15,6 @@ import Markdown
 ///
 /// A "Call to Action" has two main components: a link or file path, and the link text to display.
 ///
-/// The link path can be specified in one of two ways:
```

Comment:
Did you mean to remove this or does this PR need rebasing?

Reply:
I did mean to remove it since this information is a little redundant with the autogenerated Parameters section – but on a second look I don't think it's fully redundant so I'll add it back for now. We can restructure some of these docs in a future PR.